transport/cluster.go

```diff
@@ -177,21 +316,89 @@ func cqlRowIntoNode(r frame.Row) (*Node, error) {
 	if err != nil {
 		return nil, fmt.Errorf("rack column: %w", err)
 	}
-	tokens := r[nodeTokens] // TODO: add parsing for string list.
 	return &Node{
 		addr:       addr.String(),
 		datacenter: dc,
 		rack:       rack,
-		tokens:     tokens,
 	}, nil
 }
 
+func cqlRowIntoStrategy(r frame.Row) (*strategy, error) {
```

Comment:
Also applies to cqlRowIntoNode.
It's more idiomatic to say what you create from what i.e. readStrategyFromRow.